core/templates/dev/head/domain/question/EditableQuestionBackendApiService.ts

```diff
@@ -101,10 +102,52 @@ oppia.factory('EditableQuestionBackendApiService', [
       });
     };
 
+    var _addQuestionSkillLink = function(
+        questionId, skillId, successCallback, errorCallback) {
+      var addQuestionSkillLinkUrl = UrlInterpolationService.interpolateUrl(
+        QUESTION_SKILL_LINK_URL_TEMPLATE, {
+          question_id: questionId,
+          skill_id: skillId
+        });
+      $http.post(addQuestionSkillLinkUrl).then(function(response) {
+        if (successCallback) {
+          successCallback();
+        }
+      }, function(errorResponse) {
+        if (errorCallback) {
+          errorCallback(errorResponse.data);
+        }
+      });
+    };
+
+    var _addMultiQuestionSkillLink = function(
+        questionId, skillIds, successCallback, errorCallback) {
+      var skillIdsUrl = '';
```

Comment:
Conceptually this is not a skillIdsUrl, it is a comma separated list of skills, so call it that instead.

(Though you shouldn't be using a stringified version at all for POST -- see above.)

Ditto elsewhere.

Reply:
Done.